### DISCUSSION TITLE
how to discard the logs

### DISCUSSION THREAD

**Discussion by uglyliu** [vectordotdev/vector#17361] [Q&A] [answered]
I have the following log in my sourcelog
```
2023-05-10 httplog log log log
2023-05-10 log log log
```
How to discard the logs with httplog and not collect them

**Comment by spencergilbert** [comment, marked as answer]
You could use [remap](https://vector.dev/docs/reference/configuration/transforms/remap/) to conditionally [drop_on_abort](https://vector.dev/docs/reference/configuration/transforms/remap/#drop_on_abort), or [filter](https://vector.dev/docs/reference/configuration/transforms/filter/) those events.
